drivers/include/sx126x.h

```diff
@@ -42,6 +63,7 @@ typedef struct {
     gpio_t busy_pin;                    /**< Busy pin */
     gpio_t dio1_pin;                    /**< Dio1 pin */
     sx126x_reg_mod_t regulator;         /**< Power regulator mode */
+    sx126x_type_t type;                 /**< Variant of sx126x */
```

Comment:
Note: This extra member doesn't add ROM because of the structure alignment. Therefore I just opted to leave it as it is.